src/frontend/i18n/fr_FR.json

```diff
@@ -855,6 +855,22 @@
     "description": "",
     "message": "Remplacer cette vignette"
   },
+  "components.StudentShareDocButton.ShareDocTitleButton": {
+    "description": "Title for the upload file button",
+    "message": "Téléverser un document"
+  },
+  "components.StudentShowChatButton.ShowChatTitleButton": {
+    "description": "Title for the show/hide chat button",
+    "message": "Afficher/Masquer le chat"
+  },
+  "components.StudentQuitDiscussionButton.QuitDiscussionButton": {
+    "description": "Title for the quit discussion button",
+    "message": "Descendre de scène"
+  },
+  "components.StudentShowViewersButton.ViewersButton": {
+    "description": "Title for the show viewers button",
+    "message": "Voir les participants"
+  },
```

Comment:
@lunika should see this.
I think we handle those files later, with crowdin.

Reply:
Yes you can remove this file, It will be generated later.